File: repos_estudiantes_ALVAREZ_segunda_entrega/Calle_Castillo_Cristhian_Javier/Practica 2 prog-121/AGREGACION Y COMPOSICION/EJERCICIO 1/Habitacion/src/AgregacionYComposicion/Habitacion.java
package AgregacionYComposicion;

public class Habitacion {
    private String nombre;
    private double tamaño; 

    // Constructor
    public Habitacion(String nombre, double tamaño) {
        this.nombre = nombre;
        this.tamaño = tamaño;
    }

    // Getters
    public String getNombre() {
        return nombre;
    }

    public double getTamaño() {
        return tamaño;
    }

    // Setters
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setTamaño(double tamaño) {
        this.tamaño = tamaño;
    }

    public void mostrarInfo() {
        System.out.println("Habitación: " + nombre + ", Tamaño: " + tamaño + " m²");
    }
}

File: repos_estudiantes_TARQUI_segunda_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/COMPOSICION/EJERCICIO-1/habitacion.py
class Habitacion:
    def __init__(self, nombre, tamano):
        self.nombre = nombre
        self.tamano = tamano

    def get_nombre(self):
        return self.nombre

    def get_tamano(self):
        return self.tamano

    def set_nombre(self, nombre):
        self.nombre = nombre

    def set_tamano(self, tamano):
        self.tamano = tamano

    def mostrar_info(self):
        print(f"Habitacion: {self.nombre}, Tamaño: {self.tamano} m²")
